admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -612,30 +616,29 @@ var specificPrices = (function() {
 
   return {
     'init': function() {
-      /** set the default price to for specific price form */
-      $('#form_step2_specific_price_sp_price').val($('#form_step2_price').val());
       this.getAll();
 
-      $('#specific-price .add').click(function() {
+      $('#specific-price .add').click(function () {
```

Comment:
Rhaaaaa!

https://github.com/airbnb/javascript#functions--signature-spacing

Yep, I'm wrong, and my previous PR is all bad :(